src/modules/navigator/rtl_direct.cpp

```diff
@@ -255,8 +255,10 @@ void RtlDirect::set_rtl_item()
 				.yaw = _destination.yaw,
```

Comment:
The passed on destination already has a yaw(/heading?) which now gets ignored at least in this case here 🤯

Reply:
correct - let's go with your proposed solution.